packages/itwinui-react/src/core/Toast/Toast.test.tsx

```diff
@@ -185,3 +185,26 @@ it('should close temporary toast after 7s', () => {
 
   jest.useRealTimers();
 });
+
+it('should pass content props correctly', () => {
```

Comment:
nit: Since earlier, we were using `CommonProps` (id, className, style), it is good to test if className and style are applied correctly without `CommonProps`, like you have it now.

But, you could also test if some other props apart from those three are passed correctly. E.g., `onClick`.

This is just an optional suggestion.

Reply:
Leaving it as is for now.